spec/requests/lead_providers/npq_usage_spec.rb

```diff
@@ -21,5 +21,13 @@
 
       expect(response.body).to include("Declaring that an NPQ participant has started their induction")
     end
+
+    it "should explain how to accept an NPQ application" do
+      expect(response.body).to include("Accept an NPQ application")
```

Comment:
Missing a get here, maybe move them all to a before block?

Reply:
Cheers. I blindly copied it!